apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -334,6 +335,16 @@ export default class ScriptEditor extends React.Component {
             PDF form) that we should provide links to.
           </p>
         </label>
+        <label>
+          Text-to-Speech
+          <input
+            name="tts"
+            type="checkbox"
+            defaultChecked={this.props.tts}
+            style={styles.checkbox}
+          />
+          <p>Check if this course has text-to-speech available.</p>
```

Comment:
I think most of the time we write checkbox text as more of an action statement. "Enable text-to-speech for this unit" or something like that

Reply:
Fixed, also added screenshot.